app/routes/onboardingUser/connect.tsx

```diff
@@ -1,7 +1,7 @@
 import { FC } from 'react';
 import { ActionArgs, json, redirect } from '@remix-run/node';
-import { register } from '~/api/onboarding/register.server';
-import { FormTitlesAndSubtitles } from '~/constants/userOnboarding';
+import register from '~/api/onboarding/register.server';
```

Comment:
Same here?

Reply:
It was throwing Lint error due to which I updated it to Default export, Am I wrong here? Please check

Reply:
https://github.com/Real-Dev-Squad/website-calendar/pull/115#discussion_r1141281089